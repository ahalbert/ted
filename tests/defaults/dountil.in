beep
boop
buzz
buzz
buzz
buzz
