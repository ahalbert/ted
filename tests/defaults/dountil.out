beep
boop
boop
buzz
buzz
buzz
